### DISCUSSION TITLE
deriveClassed with react 19

### DISCUSSION THREAD

**Discussion by maltesa** [sannajammeh/tw-classed#151] [Q&A] [answered]
I was wondering if we get rid of deriveClassed with react 19 due to refs being part of the component props:
https://react.dev/blog/2024/04/25/react-19#ref-as-a-prop

Or are there other reasons to keep using it?

**Comment by sannajammeh** [comment, marked as answer]
There are other reasons to keep it, but we can remove the ref from the second argument now :)
